tests/unit/Change.spec.ts

```diff
@@ -230,7 +234,24 @@ describe('Change component', () => {
     localVue.use(VueRouter)
     const router = mockRouter.mock()
     await router.push({ name: 'change' })
-    wrapper = shallowMount(Change, { localVue, store, router, vuetify })
+    wrapper = shallowMount(Change, { localVue,
+      store,
+      router,
+      vuetify,
+      data () {
+        return { showFee: false }
+      },
+      computed: {
+        showFeeSummary: {
+          get (): boolean {
+            return this.showFee
+          },
+          set (val: boolean) {
+            this.showFee = val
+          }
```

Comment:
I think you can set state with `store.state`... instead of setting the getter explicitly. The same goes to Alteration.spec.ts.

Reply:
True for state properties. In this case, `showFeeSummary` is a computation, so he's trying to work around that by overriding/mocking it.

Ref: https://github.com/bcgov/business-edit-ui/blob/59ab050c2e617017807bc8efa055ca8406e9f2f0/src/store/getters/state-getters.ts#L831